tools/release_notes.go

```diff
@@ -0,0 +1,207 @@
+// Copyright 2021 Couchbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file  except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the  License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+)
+
+const BASE_URL = "https://issues.couchbase.com"
+
+type Issue struct {
+	Key    string      `json:"key"`
+	Fields IssueFields `json:"fields"`
+}
+
+type IssueFields struct {
+	Resolution  IssueResolution `json:"resolution"`
+	Status      IssueStatus     `json:"status"`
+	Summary     string          `json:"summary"`
+	Description string          `json:"description"`
+	Issuetype   IssueType       `json:"issuetype"`
+}
+
+type IssueType struct {
+	Name string `json:"name"`
+}
+type IssueResolution struct {
+	Name string `json:"name"`
+}
+
+type IssueStatus struct {
+	Name string `json:"name"`
+}
+type GetIssuesResponse struct {
+	Issues []Issue `json:"issues"`
+}
+
+type JiraClient struct {
+	baseUrl string
+	client  *http.Client
+}
+
+func (jc *JiraClient) GetIssues(projectName, fixVersion string) ([]Issue, error) {
+
+	reqJson, err := json.Marshal(map[string]any{
+		"jql":    "fixVersion = " + fixVersion + " AND project=" + projectName + " AND resolution != Unresolved",
+		"fields": []string{"resolution", "status", "summary", "description", "issuetype"}})
+
+	if err != nil {
+		return nil, err
+	}
+	responseBody := bytes.NewBuffer(reqJson)
```

Comment:
You're storing the requestBody in the "responseBody" variable?

Reply:
... yes. (fixed)